ui/components/app/account-menu/index.scss

```diff
@@ -105,6 +105,14 @@
     color: $white;
     padding: 3.5px 24px;
     width: 59px;
+  
+    &:hover {
+      background-color: rgba($dusty-gray, 0.05);
+    }
+
+    &:active {
+      background-color: rgba($dusty-gray, 0.1);
```

Comment:
Wondering if these should be design system colors?

Reply:
Let's merge this and save these color changes for the [colors papercut](https://www.notion.so/As-a-user-I-want-consistent-colors-used-throughout-MetaMask-aa31d68132034246bdbc0281d88cf619)(Link for internal use)